src/lib/middleware/response-time-metrics.ts

```diff
@@ -16,7 +16,7 @@ export function responseTimeMetrics(
 ): any {
     return _responseTime((req, res, time) => {
         const { statusCode } = res;
-        const pathname = req.route ? req.baseUrl + req.route.path : '(hidden)';
+        const pathname = req.route ? req.path : '(hidden)';
```

Comment:
When we have a route, they may not include a parent controller's path, i.e.:
https://github.com/Unleash/unleash/blob/d74bee472f016cf00b5b7cb0f62767f2aefe13f6/src/lib/routes/index.ts#L48-L51
And the internal route is what we get reported:
https://github.com/Unleash/unleash/blob/d74bee472f016cf00b5b7cb0f62767f2aefe13f6/src/lib/routes/proxy-api/index.ts#L87-L89